lib/collections.coffee

```diff
@@ -21,11 +21,9 @@ SimpleSchema.messages {
     label: "Serial Number"
   enteredByUserId:
     type: String
-    denyUpdate: true
```

Comment:
So, this got a little out of hand - was helping Sarah out learning Meteor + changing imported items to automatically be marked as delivered, but it looks like some of the import hooks were preventing import from working at all. Couple small issues with missing `doc`s, but also `denyUpdate` prevents setting on upserts at all (see https://github.com/aldeed/meteor-schema-deny/blob/master/package/deny/deny.js).

So, this feels kinda bad. Not sure if the change to allow/deny rules is really sufficient here, but up to you.